docs/root/start/building.rst

```diff
@@ -26,12 +26,27 @@ Building Envoy has the following requirements:
   :ref:`this FAQ entry <faq_build_speed>` for more information on build performance.
 * These :repo:`Bazel native <bazel/repository_locations.bzl>` dependencies.
 
-Please see the linked :repo:`CI <ci/README.md>` and :repo:`Bazel <bazel/README.md>` documentation
-for more information on performing manual builds.
 Please note that for Clang/LLVM 8 and lower, Envoy may need to be built with `--define tcmalloc=gperftools`
 as the new tcmalloc code is not guaranteed to compile with lower versions of Clang.
 
 
+Windows Requirements
+--------------------
+
+Envoy has been ported and is now expected to work on Windows.
+
+Building Envoy has the following requirements:
+
+* A Windows (virtual) machine running on version 1903 (10.0.18362.1) and above.
+* The Windows 10 SDK, version 1803 (10.0.17134.12). Some features may require a newer SDK.
+* `Build Tools for Visual Studio 2019 <https://visualstudio.microsoft.com/downloads/#build-tools-for-visual-studio-2019>`_
+* These :repo:`Bazel native <bazel/repository_locations.bzl>` dependencies.
+
+
+.. note::
```

Comment:
this appears to only apply to windows

Reply:
i think with this - perhaps move it to  its own section - "Further info" or somesuch

Reply:
Yeah agreed, this is what I did

Reply:
fixed in 96eea30a8107a9eb6f17af6914a4e245fcd0fc3d with `Detailed Instructions` as the section